resources/release-notes.md

```diff
@@ -41,27 +61,37 @@ this very documentation, not to a version of any APIs described by it." %}
     Seamless View][card-seamless-view] sequence diagrams.
 *   Added [Abort][swish-abort] in [Swish Payments][swish].
 
+## 29 May 2020
+
 ### Version 1.11.0
 
 *   Added a new [Trustly Payments][trustly-payments] section.
 *   Added information about [Storing URIs][storing-uri] in the [Technical
     Information][home-technical-information] section.
 
+## 22 May 2020
+
 ### Version 1.10.1
 
 *   Updated [Swish Redirect][swish-redirect] requests.
 *   Added `orderReference` information in Payment Order Callback in [Checkout Other features][checkout-callback].
 
+## 14 May 2020
+
 ### Version 1.10.0
 
 *   Added a new [Gift Cards][gift-cards] section with descriptions of our Gift Cards API.
 *   Added a new section [Co-badge Card Choice for Dankort][co-badge-card] in
     [Card Payments][card-payment-url].
 
+## 07 May 2020
+
 ### Version 1.9.2
 
 *   Corrected table in [payments page][payments]
 
+## 07 May 2020
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (87:1)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)